packages/ui/src/components/dialogs/delete-alert-dialog.tsx

```diff
@@ -77,10 +77,10 @@ export const DeleteAlertDialog: FC<DeleteAlertDialogProps> = ({
         {!!error && error.message && <p className="text-2 text-cn-foreground-danger">{error.message}</p>}
 
         <AlertDialog.Footer>
-          <Button variant="outline" onClick={onClose} disabled={isLoading}>
+          <Button variant="surface" theme="muted" onClick={onClose} disabled={isLoading}>
```

Comment:
Themes `danger`, `success` etc make sense, but `theme='muted'` is not untuitive to me, as `muted` doesn't relate to any color.

Reply:
Combined with variants like `surface` and `soft`, it's very hard to tell looking at the code what this button would look like.

Reply:
@cxramina Please check.